src/main/java/http/router/NoAuthOnRouteException.java

```diff
@@ -0,0 +1,4 @@
+package http.router;
```

Comment:
is this package used?

Reply:
yes it is.

Reply:
really? what class needs this package? i only see Throwable in this file.